modules/ecs/cluster/variables.tf

```diff
@@ -22,8 +22,13 @@ variable "log_retention" {
   description = "Cloud watch log retention days"
 }
 
-variable "allow_list" {
-  description = "IPs that are allowed to access the cluster from load balancers"
+variable "allow_list_ipv4" {
```

Comment:
This is a breaking change for the modules use in this repo, not a huge deal but a simple rename would keep things working.

Reply:
Oh yea, I missed that. 😬 Thanks for the catch!